src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+
+  if (days >= 3 && days < 7) {
+    return rentCost * days - 20;
```

Comment:
Replace the hard-coded literal '3' with a constant to represent the minimum number of days for a discount. Also, replace '20' with a constant representing the short term discount amount.

Reply:
Make changes suggested by AI